Makefile

```diff
@@ -31,10 +31,21 @@ HOST_OS := $(shell uname | tr '[:upper:]' '[:lower:]')
 export _NIX_KEEP ?= BUILD_ENV
 export NIX_CONF_DIR = $(PWD)/nix
 # We don't want to use /run/user/$UID because it runs out of space too easilly
-export TMPDIR = /tmp
+export TMPDIR = /tmp/tmp-status-react
 
 export REACT_SERVER_PORT ?= 5001 # any value different from default 5000 will work; this has to be specified for both the Node.JS server process and the Qt process
 
+#----------------
+# Required targets
+#----------------
+
+tmpdir-mk: SHELL := /bin/sh
```

Comment:
I'd prefer if this was called `_tmpdir-mk`, to make it clearer that it is not something that's supposed to be used by the end user.

Reply:
That's fair.